robottelo/host_helpers/api_factory.py

```diff
@@ -592,17 +594,17 @@ def update_provisioning_template(self, name=None, old=None, new=None):
 
         :return bool: True/False
         """
-        temp = (
+        self.temp = (
```

Comment:
Can we please change `temp` to `templ` so someone is not being misled into thinking that it is some temporary variable.

Reply:
Updated.

Reply:
@shweta83 I can't see `self.temp` renamed to `self.templ`

Reply:
also can we not squash commits so we can see changes between commits?

Reply:
@LadislavVasina1 This PR is not to be looked from nitpicks or improvements perspective but to immediatly fix the broken automation.

Reply:
Yeah I get it but changing `self.temp` to `self.temp.template` instead of `self.templ` for example is not really good change imo.